bin/main.rb

```diff
@@ -1,73 +1,73 @@
-class Player
-  attr_accessor :name, :weapon
+require '../lib/player.rb'
```

Comment:
I had to remove one dot here to make it works on my system.
Eg: `require './lib/player.rb'`
It works for you that way?

Reply:
Well it tested it on Ubuntu and it worked with the '../lib/player.rb'